app/host/tests/test_ebv.py

```diff
@@ -0,0 +1,33 @@
+import numpy as np
+from django.test import TestCase
+
+from ..models import Transient
+from ..transient_tasks import MWEBV_Host
+from ..transient_tasks import MWEBV_Transient
+
+
+class EBVTest(TestCase):
+    fixtures = [
+        "../fixtures/initial/setup_survey_data.yaml",
+        "../fixtures/initial/setup_filter_data.yaml",
+        "../fixtures/initial/setup_catalog_data.yaml",
+        "../fixtures/initial/setup_status.yaml",
+        "../fixtures/initial/setup_tasks.yaml",
+        "../fixtures/initial/setup_acknowledgements.yaml",
+        "../fixtures/example/2010h.yaml",
```

Comment:
Please copy and freeze "../fixtures/example/2010h.yaml" to "../fixtures/test/test_2010h.yaml" and refer to it here. I want your test to pass forever, and not fail if we change the example :)